rhv_checks.py

```diff
@@ -338,6 +340,54 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, unknown, all_items = [], [], [], []
+    command = "hosted-engine --vm-status --json"
+    hosts = system.api.system_service().hosts_service().list()
+    output = ssh_command(hosts[0].name, password, command)
```

Comment:
How are we passing password to the check? @john-dupuy

Reply:
@sbulage How are you passing the password here?

Reply:
@kedark3 @john-dupuy How are passing password?, I have raised this concern in Infra meeting @kedark3 told we can discuss this in call.